### PR TITLE
Removed substr call in amqp routing key generator

### PR DESCRIPTION
**Pull request by acrobat**
Fixed todo from #197

> Remove substr TODO in AmqpHandler

### CODE REVIEW

File: src/Monolog/Handler/AmqpHandler.php [hunk 1]
```diff
@@ -58,8 +58,7 @@ protected function write(array $record)
 
         $routingKey = sprintf(
```
Comment (dosten) [inline src/Monolog/Handler/AmqpHandler.php]: IMO this should be in a single line

Comment (acrobat) [inline src/Monolog/Handler/AmqpHandler.php]: Fixed!

**Comment by acrobat** [comment]
Rebase branch and changed fix after changes in master

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**